modules/conflict_resolver/jsx/conflictResolverIndex.js

```diff
@@ -0,0 +1,293 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import Loader from 'Loader';
+import {Tabs, TabPane} from 'Tabs';
+import FilterForm from 'jsx/FilterForm';
+import DynamicDataTable from 'jsx/DynamicDataTable';
+/**
+ * This file contains the React classes for conflict resolver
+ * module.
+ */
+
+/**
+ * This is the React class for Unresolved Conflicts
+ */
+class UnresolvedConflictsPane extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      Data: this.props.data,
+    };
+    this.formatColumn = this.formatColumn.bind(this);
+  }
+
+  updateFilterState(filter) {
+    this.setState({filter});
+  }
+
+  formatColumn(column, cell, rowData, rowHeaders) {
+    if (loris.hiddenHeaders.indexOf(column) > -1) {
```

Comment:
even though I think you can use hidden headers here, we shouldn't be using the loris global variable to get it. instead, define a state 'hiddenHeaders' as an array of the headers to be hidden and then use this.state.hiddenHeaders here. you can find an example in ImagingUploader:

<img width="376" alt="Screenshot 2019-03-12 19 19 10" src="https://user-images.githubusercontent.com/17415878/54242674-dc005480-44fb-11e9-9b2e-a6d08bbacf13.png">